server/models/handlers/input_schedule.js

```diff
@@ -0,0 +1,15 @@
+// Create stub to input weekly worker schedule into database 
```

Comment:
Also make sure to implement the router so the front end knows what endpoint to call. E.g. `api/login`. Refer to routes folder, particularly public/private file for examples. Your endpoint will most likely be in the private file.